kyuubi-hive-beeline/src/main/resources/BeeLine.properties

```
@@ -154,73 +154,76 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
+\                                    Setting --incremental=true is recommended if you encounter an OutOfMemory\n \
+\                                    on the client side (due to the fetched result set size being large).\n \
+\                                    Only applicable if --outputformat=table.\n\n \
+\  --incrementalBufferRows=NUMROWS   The number of rows to buffer when printing rows on stdout,\n \
+\                                    defaults to 1000; only applicable if --incremental=true\n \
+\                                    and --outputformat=table.\n\n \
+\  --truncateTable=[true|false]      Truncate table column when it exceeds length.\n \
+\  --delimiterForDSV=DELIMITER       Specify the delimiter for delimiter-separated values output format (default: |).\n \
+\  --isolation=LEVEL                 Set the transaction isolation level.\n \
+\  --nullemptystring=[true|false]    Set to true to get historic behavior of printing null as empty string.\n \
+\  --maxHistoryRows=MAXHISTORYROWS   The maximum number of rows to store beeline history.\n \
+\  --delimiter=DELIMITER             Set the query delimiter; multi-char delimiters are allowed, but quotation\n \
+\                                    marks, slashes, and -- are not allowed; defaults to ;\n\n \
+\  --convertBinaryArrayToString=[true|false]\n \
+\                                    Display binary column data as string or as byte array.\n\n \
+\  --python-mode                     Execute python code/script.\n \
+\  --help                            Display this message.\n \
 \n \
 \  Example:\n \
-\   1. Connect using simple authentication to HiveServer2 on localhost:10000\n \
-\   $ beeline -u jdbc:hive2://localhost:10000 username password\n\n \
-\   2. Connect using simple authentication to HiveServer2 on hs.local:10000 using -n for username and -p for password\n \
-\   $ beeline -n username -p password -u jdbc:hive2://hs2.local:10012\n\n \
-\   3. Connect using Kerberos authentication with hive/localhost@mydomain.com as HiveServer2 principal\n \
-\   $ beeline -u "jdbc:hive2://hs2.local:10013/default;principal=hive/localhost@mydomain.com"\n\n \
-\   4. Connect using SSL connection to HiveServer2 on localhost at 10000\n \
-\   $ beeline "jdbc:hive2://localhost:10000/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"\n\n \
+\   1. Connect using simple authentication to KyuubiServer on localhost:10009\n \
+\   $ kyuubi-beeline -u jdbc:kyuubi://localhost:10009 username password\n\n \
+\   2. Connect using simple authentication to KyuubiServer on kyuubi.local:10009 using -n for username and -p for password\n \
+\   $ kyuubi-beeline -n username -p password -u jdbc:kyuubi://kyuubi.local:10009\n\n \
+\   3. Connect using Kerberos authentication with hive/localhost@mydomain.com as KyuubiServer principal\n \
+\   $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;principal=hive/localhost@mydomain.com"\n\n \
```

actually, the `principal` is a confusing name, I see lots of users new to kerberos think it is the client principal name. we made some improvements on Kyuubi JDBC driver on this area, see https://kyuubi.readthedocs.io/en/master/client/jdbc/kyuubi_jdbc.html
```suggestion
\   3. Connect using Kerberos authentication with `kyuubi/localhost@mydomain.com` as KyuubiServer principal (kinit is required before connection)\n \
\   $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"\n\n \
```

And Kyuubi also supports using keytab for Kerberos authentication directly (without required to perform kinit to generate TGT cache)

```
kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;kyuubiClientPrincipal=user@mydomain.com;kyuubiClientKeytab=/local/path/client.keytab;kyuubiServerPrincipal=kyuubi/localhost@mydomain.com"
```